_includes/footer.html

```diff
@@ -1,44 +1,35 @@
-
 <footer class="page-footer">
   <div class="container">
-    <div class="row">
-      <div class="col l6 s12">
-        <h5 class="white-text">viperdev</h5>
-        <p class="grey-text text-lighten-4">The Leading Edge Way to Improve Code Quality</p>
-
-        <br>
-        <span style="color:whitesmoke">Navigation</span>
-
-        <ul class="footer-nav-links">
-          {% assign sorted_pages = site.pages | sort:"order" %}
-          {% for node in sorted_pages %}
-          {% if node.order %}
-          {% if page.url == node.url %}
-          <li class="active"><a href="{{node.url | prepend: site.baseurl }}" class="active">{{node.title}}</a></li>
-          {% else %}
-          <li><a href="{{node.url | prepend: site.baseurl }}">{{node.title}}</a></li>
-          {% endif %}
-          {% endif %}
-
-          {% endfor %}
-        </ul>
-      </div>
-      <div class="col l4 offset-l2 s12">
-        <h5 class="white-text">Any Questions or Queries ?</h5>
-
-<div >
- <a href="https://gitter.im/viper-development/support" target="blank" class="mailbtn btn-large waves-effect grey-text text-lighten-4 activate-sidecar">Chat with us <span class=""><i class="mailicn fa fa-commenting" aria-hidden="true"></i>
-</span></a>
+  <div class="row">
+  <div class="col-md-8 s12">
```

Comment:
this seems a functional change and not only codestyle, also why are you messing up indentation for this?